server.js

```diff
@@ -116,10 +126,21 @@ function handleSocketConnection(socket) {
     cb(EXTRA_VALID_WORDS_REGEXP.test(word) || dictionary.check(word))
   })
 
+  socket.on('vote', function tallyVote(vote) {
+    if (roundManager.state === 'vote') {
+      users.vote(vote)
+      votesNeeded -= 1
```

Comment:
this is 100% fine as-is, but you can also use `--votesNeeded` and `++foobar` in JS, in case you weren't aware. Just as an fyi